### DISCUSSION TITLE
storage folder should be copied from dev path to dist on build process

### DISCUSSION THREAD

**Discussion by rince1984** [neutralinojs/neutralinojs#544] [Ideas]
Actually after build process (CLI: neu build) the storage folder does not exist in the dist folder. 
It would be nice if the storage folder would be created automatically through build process (in the dist folder). Much better would be a copy of the existing storage folder from app path itself. So I could store default values in it before build process. In that case the storage folder would only be copied if it exists in the app path (development path).

**How I came to this:**
I have an app where I need to store global settings in the storage. On app start I check the storage for a setting. If the storage entry is set to true I will display a modal window on app start. That works great in development. But after run the CLI build process ```neu build``` the .storage folder does not exist and I get an error.

**Comment by shalithasuranga** [comment]
Hey.. Thanks for the idea. now you can use `neu build --copy-storage`

**Reply by rince1984** [reply]
You are the best!
